src/main/resources/db/migration/V1_13__Create_case_data_tables.sql

```diff
@@ -7,12 +7,13 @@ create table if not exists darts.moj_reporting_restrictions
 create sequence if not exists moj_rer_seq;
 
 
-create table if not exists darts.moj_case
+create table if not exists  moj_case
 (
   moj_cas_id         integer not null primary key,
   moj_rer_id         integer
-    constraint moj_case_reporting_restriction_fk references darts.moj_reporting_restrictions (moj_rer_id),
+    constraint moj_case_reporting_restriction_fk references  moj_reporting_restrictions (moj_rer_id),
   r_case_object_id   varchar(16),
+  c_type                    CHARACTER VARYING,
```

Comment:
The LLD said we don't need c_type.  Barry confirmed.

Reply:
removed